packages/_private/angular-demo/src/app/themeswitch/themeswitch.component.ts

```diff
@@ -0,0 +1,32 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'theme-switch',
+  styleUrls: ['./themeswitch.css'],
+  templateUrl: './themeswitch.component.html',
+})
+export class ThemeSwitchComponent {
+
+  buttonText: string = 'Switch Theme';
+
+  switchTheme() {
+    const bdy = document.body;
```

Comment:
Could you name it `body` instead of `bdy` or is this a forbidden word? We don't use many abbreviations a lot, so I would avoid them...